build/docker/milvus/ubuntu20.04/Dockerfile

```diff
@@ -14,8 +14,11 @@ FROM milvusdb/openblas:ubuntu20.04-20220914-179ea77 AS openblas
 #FROM alpine
 FROM ubuntu:focal-20220426
 
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=UTC
+
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends libtbb-dev gfortran netcat iputils-ping ca-certificates liblapack3 && \
+    apt-get install -y --no-install-recommends curl libtbb-dev gfortran netcat iputils-ping ca-certificates liblapack3 libzstd-dev uuid-dev libaio-dev libgoogle-perftools-dev clang-format libboost-all-dev && \
```

Comment:
libgoogle-perftools-dev clang-format are unnecessary dependencies for diskAnn , please check the pr https://github.com/milvus-io/knowhere/pull/470/files

Reply:
also use libboost-program-options-dev instead of libboost-all-dev , it will reduce size ,check the pr for knowhere https://github.com/milvus-io/knowhere/pull/472/files#diff-0eec4d3929920fcde515122c9e2f7711593024b14f6c59970ba4a1f00842a4e8L5

Reply:
Done